src/test/java/uk/gov/hmcts/ccd/data/casedataaccesscontrol/DefaultRoleAssignmentRepositoryIT.java

```diff
@@ -238,6 +243,128 @@ private void validateRoleAssignment(RoleAssignmentResource actualRole,
     }
 
 
+    @Nested
+    @DisplayName("deleteRoleAssignmentsByQuery()")
+    class DeleteRoleAssignmentsByQuery {
+
+        private StubMapping badRasStub;
+        private final String deleteUrl = "/am/role-assignments/query/delete";
+
+        private final String caseId1 = "11111";
+        private final String caseId2 = "22222";
+
+        private final String userId1 = "12345";
+        private final String userId2 = "23456";
+
+        private final String role1 = "[ROLE1]";
+        private final String role2 = "[ROLE2]";
```

Comment:
If these are constants, should they be static ?

Reply:
Resolved in 1ee8ce11700eb2cfa7b7763bbd00bd9846172417.